src/Sulu/Bundle/AdminBundle/Resources/js/components/Checkbox/CheckboxGroup.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+import React from 'react';
+import type {ChildrenArray, Element} from 'react';
+import Checkbox from './Checkbox';
+
+type Props<T> = {|
+    children: ChildrenArray<Element<typeof Checkbox>>,
+    className?: string,
+    disabled: boolean,
+    onChange: (values: Array<T>) => void,
+    values: Array<T>,
+|};
+
+export default class CheckboxGroup<T: string | number> extends React.PureComponent<Props<T>> {
+    static defaultProps = {
+        disabled: false,
+    };
+
+    handleChange = (checked: boolean, changedValue: ?T) => {
+        const {onChange, values} = this.props;
+
+        if (checked && changedValue) {
+            onChange([...values, changedValue]);
+        } else {
+            onChange(values.filter((value) => value !== changedValue));
+        }
+    };
+
+    render() {
+        const {className, disabled, values} = this.props;
+
+        return (
+            <div className={className}>
+                {React.Children.map(this.props.children, (child) => {
+                    return React.cloneElement(child, {
+                        checked: !!values && values.includes(child.props.value),
```

Comment:
does flow expect the `!!values` check here? because this should always be an array according to the props type

Reply:
TBH I just copied this component from somewhere and adjusted it... Seems like this is really not necessary :slightly_smiling_face: